lib/models/product.dart

```diff
@@ -3,7 +3,7 @@ import 'package:cloud_firestore/cloud_firestore.dart';
 
 class Product {
   final String id;
-  final String renter;
+  final String uploader;
   final String title;
```

Comment:
Personally I think renter is a better description, but as discussed at the meetings the uploader could be a different person than the actual renter, so let's go with uploader for now.